packer/studio_ami.pkr.hcl

```diff
@@ -11,9 +11,10 @@ variables {
   image_name             = "studio-selfhosted"
   image_description      = "Iterative Studio Selfhosted - {{isotime `2006-01-02`}}"
   aws_build_region       = "us-west-1"
-  aws_build_instance     = "m6i.large"
+  aws_build_instance     = "c6a.large"
```

Comment:
Last month in both zones:
![image](https://user-images.githubusercontent.com/5058132/234260336-4802d4f9-f262-40e9-bcdf-7c1b27234efb.png)
![image](https://user-images.githubusercontent.com/5058132/234260406-3af351e8-ab8d-4116-afd2-8a542b37f759.png)